### PR TITLE
feat: added cart page and updated styling for project

### PR DESCRIPTION
**Pull request by sharath-io**
deploy preview:  https://deploy-preview-6--meta-store.netlify.app/pages/cart.html

Screen Shot 2022-02-21 at 11 25 23 PM

### CODE REVIEW

File: css/project.css [hunk 1]
```diff
@@ -5,30 +5,22 @@
 
 :root {
     --secondary-background: #FFFFFF;
+    --login-color: #2455f4;
 }
```
Comment (Rockyraven) [inline css/project.css:8]: use there var color

**Review by Rockyraven — APPROVED** [review]
use var color
after that all goods to me go ahead

File: css/project.css (hunk 1 above)
Comment (sharath-io) [reply css/project.css:8]: You mean instead of login-color should i use something like secondary-color

**Review by NeerajVodala — APPROVED** [review]
Looks good. All the best.

File: css/project.css (hunk 1 above)
Comment (sharath-io) [reply css/project.css:8]: updated with css variable

**Pull request merged**